view/adminhtml/templates/system/loimage.phtml

```diff
@@ -0,0 +1,12 @@
+<div class="<?php echo $block->getClass() ?>">
+    <p>
+        <?php echo $block->getMarkup() ?>
+    </p>
+    <?php
+        if ($block->getImgurl()) {
+            echo '<div style="text-align:center;">';
+            echo '<img src="' . $block->getImgurl() . '" style="width:75%;">';
+            echo '</div>';
+        }
+    ?>
+</div>
```

Comment:
If you close the PHP block that has the if, you can actually just write real HTML rather than echoing it.

```php
<?php if ($block->getImgurl()): ?>
<div>
...
</div>
<?php endif; ?>
```

Sample from M2 codebase: https://github.com/magento/magento2/blob/2.3-develop/setup/view/error/index.phtml

Forget the technical name for this syntax offhand...